src/index.ts

```diff
@@ -111,8 +124,11 @@ export function restoreResumableFields(id: string, options?: RestoreOptions): vo
 
     if (document.dispatchEvent(resumeEvent)) {
       const field = document.getElementById(fieldId)
-      if (field && (field instanceof HTMLInputElement || field instanceof HTMLTextAreaElement)) {
-        if (field.value === field.defaultValue) {
+      if (isPersistElement(field)) {
+        if (field instanceof HTMLSelectElement) {
+          field.value = value
+          changedFields.push(field)
+        } else if (field.value === field.defaultValue) {
```

Comment:
I realize it would probably complicate things a lot so I'm okay with this gap for now, but I think it would be possible to update multi selects in a loop. Something like

```ts
const selected = ['option 1', 'option 2']

for(const option of Array.from(field.options)) {
    option.selected = selected.includes(option.value)
}
```

Reply:
Maybe it would be good to add a note for now that this gap exists... and maybe an issue to close it?